src/integration-test/tests/citizen/claim/helpWithFees_test.ts

```diff
@@ -9,11 +9,6 @@ Scenario('Navigate Payment Page by providing claim details... @smoke-test', { re
   await claimSteps.makeAClaimAndNavigateUpToPayment(I)
 })
 
-// The @citizen-smoke-test tag used for running smoke tests with pre-registered user for help with fees
```

Comment:
Removing this test as the same steps are executed in 
Scenario('Submit Claim via HWF Reference... @citizen', { retries: 3 }, async (I: I) => {
  await claimSteps.makeAHwfClaimAndSubmit(I)
})